### PR TITLE
chore: update HorizontalPodAutoscaler apiVersion

### PR DESCRIPTION
**Pull request by quarckster**
Supersedes #67.

### CODE REVIEW

File: haproxy-ingress/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This chart bootstraps an HAProxy Ingress deployment on a [Kubernetes](http://kub
 ## Prerequisites
 
 * Helm 3 installed, see installation instructions [here](https://helm.sh/docs/intro/install/)
-* Kubernetes 1.8+ with Beta APIs enabled
+* Kubernetes 1.23+ with Beta APIs enabled
```
Comment (jcmoraisjr) [inline haproxy-ingress/README.md:12]: You can change this to 1.19 instead, which's now the ingress api limitation.

**Review by jcmoraisjr — COMMENTED** [review]
Thanks! Sorry for the long delay. Just a small suggestion and we're done.

File: haproxy-ingress/README.md (hunk 1 above)
Comment (quarckster) [reply haproxy-ingress/README.md]: Done.

**Comment by jcmoraisjr** [comment]
thanks! lgtm, merging now

**Pull request merged**